service/device/registry/test-steps/src/main/java/org/eclipse/kapua/service/device/registry/steps/DeviceRegistrySteps.java

```diff
@@ -2559,4 +2560,65 @@ public void iFindDeviceWithClientId(String deviceName) {
 
         assertEquals(device.getClientId(), deviceName);
     }
+
+    @And("^I add device to group$")
+    public void iAddDeviceToGroup() throws Exception{
+        Device device = (Device) stepData.get("Device");
```

Comment:
Change method implementation, so it accepts the parameters that are given to it; do not use "stepData.get("Device")